ui/ui-components/pages/object/[id].tsx

```diff
@@ -54,10 +54,24 @@ export default function FindObject(props) {
       response.records.length === 1 &&
       grouparooUiEdition() === "community"
     ) {
-      const listPage = getListPage(response.records[0].tableName.toLowerCase());
-      router.push(listPage);
+      const tableName = response.records[0].tableName.toLowerCase();
```

Comment:
Note that there's an incoming change in this area here https://github.com/grouparoo/grouparoo/pull/2783